Makefile

```diff
@@ -316,3 +316,12 @@ ci-get-version-package:
 .PHONY: ci-prepare-legal
 ci-prepare-legal:
 	@$(CURDIR)/scripts/ci-helper.sh prepare-legal
+
+.PHONY: ci-copywriteheaders
+ci-copywriteheaders:
+	copywrite headers --plan
+	# Special case for MPL headers in /api, /sdk, and /shamir
+	cd api && $(CURDIR)/scripts/copywrite-exceptions.sh
+	cd sdk && $(CURDIR)/scripts/copywrite-exceptions.sh
+	cd shamir && $(CURDIR)/scripts/copywrite-exceptions.sh
```

Comment:
It seems that `copywrite-excetions.sh` does indeed apply the changes. This is different that what the one in line 322 does. Should they be both do the same thing?

Reply:
Thanks for catching this, it was an accident, I've updated all copywrite invocations to use the --plan flag.